packages/edit-site/src/components/page-templates/index.js

```diff
@@ -252,10 +256,10 @@ export default function DataviewsTemplates() {
 		[ authors, view.type ]
 	);
 
-	const { shownTemplates, paginationInfo } = useMemo( () => {
+	const { data, paginationInfo } = useMemo( () => {
```

Comment:
Can this hook become a reusable hook that filters and sorts and everything based on "view", "allData" and "fields"?

Reply:
We can explore this separately better because I think we need to advance the filters API more and see how we can abstract it best. Right now there is a lot of specific code regarding the filtering(see author in templates).

Reply:
It doesn't seem specific at all for me. `author_text` is just `getValue`  and the rest is standard. I'm fine if it's explored separately but I think having something like this is going to help us make the right decisions. And we won't need to expose multiple helpers from the dataviews package only a single hook.